build-settings-logic/settings.gradle.kts

```diff
@@ -0,0 +1,29 @@
+import org.gradle.api.initialization.resolve.RepositoriesMode.PREFER_SETTINGS
```

Comment:
imports should be below copyright (IDEA auto-import for some reason do it otherwise in `build-scripts`)